pkg/reconciler/revision/table_test.go

```diff
@@ -549,6 +549,30 @@ func TestReconcile(t *testing.T) {
 			Object: pa("foo", "pull-backoff", WithReachabilityUnreachable),
 		}},
 		Key: "foo/pull-backoff",
+	}, {
+		Name: "surface ImagePullBackoff when previously scaled ok but now image is missing",
```

Comment:
This covers the case where we are not in a rolout eg. scaling from zero and image is not there.